### ISSUE TITLE
FilenameIndex.getVirtualFilesByName: Assertion failed

### ISSUE THREAD

**Issue by xMichaelSilvax** [wt-io-it/odoo-pycharm-plugin#110]
Plugin Version: 0.6.9
IntelliJ Version: PyCharm 2022.3
IntelliJ Build: PY-223.7571.203

<details><summary>java.lang.Throwable: Assertion failed</summary>

```
java.lang.Throwable: Assertion failed
	at com.intellij.openapi.diagnostic.Logger.assertTrue(Logger.java:254)
	...
	at com.intellij.psi.search.FilenameIndex.getVirtualFilesByName(FilenameIndex.java:62)
	at at.wtioit.intellij.plugins.odoo.compatibility.CompatibleFileIndex.getVirtualFilesByName(CompatibleFileIndex.java:44)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.lambda$getModuleDirectorySlow$5(OdooModuleServiceImpl.java:149)
	at at.wtioit.intellij.plugins.odoo.WithinProject.call(WithinProject.java:12)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.lambda$getModuleDirectorySlow$6(OdooModuleServiceImpl.java:148)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:860)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.getModuleDirectorySlow(OdooModuleServiceImpl.java:144)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.getModuleDirectory(OdooModuleServiceImpl.java:131)
	at at.wtioit.intellij.plugins.odoo.modules.index.OdooDeserializedModuleImpl.getDirectory(OdooDeserializedModuleImpl.java:64)
	at at.wtioit.intellij.plugins.odoo.WithinProject.call(WithinProject.java:15)
	at at.wtioit.intellij.plugins.odoo.pycharm.PyCharmOdooAddonsImportResolver.resolveImportReference(PyCharmOdooAddonsImportResolver.java:50)
	at com.jetbrains.python.psi.resolve.PyResolveImportUtil$foreignResults$1.invoke(PyResolveImportUtil.kt:199)
	at com.jetbrains.python.psi.resolve.PyResolveImportUtil$foreignResults$1.invoke(PyResolveImportUtil.kt:199)
	at kotlin.sequences.TransformingSequence$iterator$1.next(Sequences.kt:210)
	...
	at kotlin.sequences.SequencesKt___SequencesKt.toList(_Sequences.kt:808)
	at com.jetbrains.python.psi.resolve.PyResolveImportUtil.foreignResults(PyResolveImportUtil.kt:201)
	at com.jetbrains.python.psi.resolve.PyResolveImportUtil.resolveQualifiedName(PyResolveImportUtil.kt:74)
	at dev.ngocta.pycharm.odoo.python.OdooCanonicalPathProvider.getCanonicalPath(OdooCanonicalPathProvider.java:28)
	at com.jetbrains.python.psi.resolve.QualifiedNameFinder.canonizeQualifiedName(QualifiedNameFinder.java:182)
	...
	at com.jetbrains.python.codeInsight.completion.PyModuleNameCompletionContributor.fillCompletionVariants(PyModuleNameCompletionContributor.kt:52)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributor(CompletionService.java:81)
	...
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:246)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	...
	at java.base/java.lang.Thread.run(Thread.java:833)
```
</details>

**Comment by bigbear3001** [comment]
The fix from #33 seems no longer working for 2022.3

**Comment by ap-wtioit** [comment]
This should also be fixed with #67 in 0.6.10
